src/Acuminator/Acuminator.Utilities/Roslyn/Semantic/IMethodSymbolExtensions.cs

```diff
@@ -347,5 +347,18 @@ public static bool IsDefinitelyStatic(this IMethodSymbol method, SyntaxNode meth
 
 			return methodDeclaration.IsStatic();
 		}
+
+		/// <summary>
+		/// Check if <paramref name="method"/> has either virtual or override signature.
+		/// </summary>
+		/// <param name="method">The method to act on.</param>
+		/// <returns>True if <paramref name="method"/> </returns>
+		public static bool CanBeOverriden(this IMethodSymbol method)
+		{
+			method.ThrowOnNull(nameof(method));
+
+			return method.IsVirtual || method.IsOverride;
```

Comment:
`abstract` members can also be overridden, should we also check for it?

Reply:
Yes. I checked and Roslyn considers abstract methods to be non virtual, so we should check `IsAbstract` flag too. Thanks for noticing.

@taskinkemal please add a check for `IsAbstract` method:
```suggestion
			return method.IsVirtual || method.IsOverride || method.IsAbstract;
```

Reply:
Sure, done. Thanks.